src/coreclr/CMakeLists.txt

```diff
@@ -5,13 +5,15 @@ cmake_policy(SET CMP0042 NEW)
 # Set the project name
 project(CoreCLR)
 
+include(${CLR_ENG_NATIVE_DIR}/configureplatform.cmake)
+
 # Include cmake functions
 include(${CLR_ENG_NATIVE_DIR}/functions.cmake)
 
-if (WIN32)
+if (CLR_CMAKE_TARGET_WIN32)
   message(STATUS "VS_PLATFORM_TOOLSET is ${CMAKE_VS_PLATFORM_TOOLSET}")
   message(STATUS "VS_PLATFORM_NAME is ${CMAKE_VS_PLATFORM_NAME}")
-endif (WIN32)
+endif (CLR_CMAKE_TARGET_WIN32)
```

Comment:
These messages are about the compiler toolset.  These should be configured by `CLR_CMAKE_HOST_*` variables.